sdc/tests/test_io.py

```diff
@@ -72,8 +72,30 @@ def setUp(self):
             A = np.random.ranf(n)
             A.tofile("np_file1.dat")
 
-    @unittest.skip('Error - fix needed\n'
-                   'NUMA_PES=3 build')
+    def assertDataEqual(self, left, right):
```

Comment:
Let's support passing of extra arguments through `*args` and `**kwargs`. Maybe it will be required.

Reply:
@densmirn could you propose the modification?

Reply:
E.g. let's consider parameter `check_names=False`. You could pass the parameter to assert via `kwargs`. You could make as below:
```
def assertDataEqual(self, left, right, *args, **kwargs):
    # ...
    pd.testing.assert_series_equal(left, right, *args, check_categorical=False, **kwargs)
    # ...
```